### PR TITLE
Adding an ability to register activities and workflows in specific workers

### PR DESCRIPTION
**Pull request by msmakouz**
| Q             | A
| ------------- | ---
| Bugfix?       | ❌
| Breaks BC?    | ❌ 
| New feature?  | ✔️

- Added an ability to register activities and workflows in specific workers.
Need to add `Spiral\TemporalBridge\Attribute\AssignWorker` with a worker name.
```php
use Spiral\TemporalBridge\Attribute\AssignWorker;

#[AssignWorker(name: 'worker1')]
class SomeActivity
{
}
```
- Added `defaultWorker` parameter in config. In this parameter, can specify the name of the default worker.
- Added  `workers` parameter in config. In this parameter, can set worker options.
```php
// file app/config/temporal.php

use Temporal\Worker\WorkerOptions;

return [
    'workers' => [
        'worker1' => WorkerOptions::new()
    ]
]
```
- Added method `addWorkerOptions` to the `Spiral\TemporalBridge\Bootloader\TemporalBridgeBootloader`. This method allows to register workers with the necessary options.
```php
<?php

declare(strict_types=1);

namespace App\Bootloader;

use Spiral\Boot\Bootloader\Bootloader;
use Spiral\TemporalBridge\Bootloader\TemporalBridgeBootloader;
use Temporal\Worker\WorkerOptions;

final class AppBootloader extends Bootloader
{
    public function boot(TemporalBridgeBootloader $temporalBridgeBootloader): void 
    {
        $temporalBridgeBootloader->addWorkerOptions('first', WorkerOptions::new());
        $temporalBridgeBootloader->addWorkerOptions('second', WorkerOptions::new());
    }
}
```

### CODE REVIEW

File: src/Dispatcher.php [hunk 1]
```diff
@@ -36,13 +39,19 @@ public function serve(): void
...
+        $defaultWorker = $this->container->get(TemporalConfig::class)->getDefaultWorker();
+        $reader = $this->container->get(ReaderInterface::class);
 
         foreach ($declarations as $type => $declaration) {
+            $registerWorker = $reader->firstClassMetadata($declaration, RegisterWorker::class);
+
+            // Worker that listens on a task queue and hosts both workflow and activity implementations.
+            $worker = $registry->get($registerWorker === null ? $defaultWorker : $registerWorker->name);
+
+            $worker->registerActivityFinalizer(fn() => $finalizer->finalize());
```
Comment (butschster) [inline src/Dispatcher.php]: Maybe we move it into Registry?

**Review by butschster — APPROVED** [review]

**Comment by butschster** [comment]
Let's add  `addWorkerOptions` method for the bootloader?

```php
TemporalBridgeBootloader::addWorkerOptions('worker2', WorkerOptions::new());
```

**Comment by butschster** [comment]
- [x] Add infromation about configuration into Readme
- [x] Information about RegisterWorker attributte

File: src/Dispatcher.php (hunk 1 above)
Comment (msmakouz) [reply src/Dispatcher.php]: Moved

**Review by butschster — APPROVED** [review]

**Comment by butschster** [comment]
@msmakouz Please rename `RegisterWorker` -> `AssignWorker`

**Pull request merged**